pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools", "numpy>=1.8", "cython"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "powerfit"
+version = "2.0.0"
+description = "Rigid body fitting of high-resolution structures in low-resolution cryo-electron microscopy density maps"
+authors = [
+    {name = "Gydo C.P. van Zundert", email = "g.c.p.vanzundert@uu.nl"},
+]
+license = "Apache-2.0"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python :: 3.10",
```

Comment:
Can we go straight to 3.13?

Reply:
I am still running Ubuntu 22.04 which ships with 3.10, that is why.

I have not looked at opencl to see how hard it depends on the OS python. 
If that dependency is loose enough we can do >=3.13 and I can use uv or conda to get 3.13.

On the other hand. Where do we expect powerfit to be installed? If some random user does `pip install powerfit` then I would like to support more Python versions.

Reply:
Yep I would like to have this over pypi with the other applications - so indeed random pip install would be the case. But we can tackle the python versions after this one is done